bin/icons/config.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path')
+
+const { cosmiconfig } = require('cosmiconfig')
+
+const defaultFilepath = path.resolve(__dirname, '../../config.js')
+const defaultConfig = {
+  icons: [],
+
+  output: {
+    path: path.resolve(__dirname, '../../lib'),
+    publicPath: '/',
+    icons: {
+      files: null,
+      sprite: null,
+    },
+  },
+
+  svgo: {
+    plugins: {
+      removeViewBox: false,
+      removeDimensions: true,
+    },
+  },
+}
+
+const doc = {
+  icons:
+    'config.icons should be either string, array or an object { files, path }',
+}
+
+const read = async (filepath) => {
+  const explorer = cosmiconfig('design')
+  if (filepath != null) return explorer.load(filepath)
+
+  let config = await explorer.search()
+  if (config == null) config = await explorer.load(defaultFilepath)
+
+  return config
+}
```

Comment:
```suggestion
const read = async (filepath = defaultFilepath) => {
  const explorer = cosmiconfig('design')
  return explorer.load(filepath)
}
```

Reply:
It's not like that. I want to load the default `design` config only if there is not project-specific one.